ol7/checks/oval/file_permissions_unauthorized_suid.xml

```diff
@@ -0,0 +1,93 @@
+<def-group>
+  <definition class="compliance" id="file_permissions_unauthorized_suid" version="1">
```

Comment:
Same comment - could we use shared here?

Reply:
No shared corresponding OVAL check observed at the moment. Could be shared in case  'var_suid_whitelist' which is product specific defined outside of this rule.